FINE/utils.py

```diff
@@ -598,59 +596,24 @@ def output(output, verbose, val):
     if verbose == val:
         print(output)
 
-
 def checkModelclassEquality(esM, file):
-
-    def compareLists(list1,list2):
-        # Check if two given lists have the same items.
-        if len(list1) != len(list2):
-            return False
-        for item in list1:
-            if list1.count(item) != list2.count(item):
-                break
-        else: # else clause of the for-loop
-            return True
-        return False
-
     mdlList = list(esM.componentModelingDict.keys())
     compList = []
-
-    for cl in mdlList:
-        x = cl[0:len(cl)-5]
-        r = re.compile(x)
-        test = list(filter(r.match, file.sheet_names))
-        if len(test)>0:
-            compList.append(cl)
-
-    if not compareLists(mdlList, compList):
+    for sheet in file.sheet_names:
+        compList += [cl for cl in mdlList if (cl[0:-5] in sheet and cl not in compList)]
+    if len(mdlList) != len(compList):
         raise ValueError('Loaded Output does not match the given energy system model.')
-
+    for item in compList:
+        if compList.count(item) != mdlList.count(item):
+            raise ValueError('Loaded Output does not match the given energy system model.')
```

Comment:
Hi Theresa,
I think you can shorten these lines by writing: if set(mdlList) == set(compList): ...